.env.example

```diff
@@ -15,6 +15,11 @@ HEADLESS="true"
 # at the same brokerage with a comma, then separate account credentials with a colon
 # BROKER=BROKER_USERNAME:BROKER_PASSWORD,OTHER_BROKER_USERNAME:OTHER_BROKER_PASSWORD
 
+# dSPAC Pro
+# USE_EMAIL takes either TRUE or FALSE. TRUE mean email for username, FALSE means phone number
+# DSPAC=DSPAC_USERNAME:DSPAC_PASSWORD:USE_EMAIL TRUE/FALSE
+DSPAC=
+
```

Comment:
As NelsonDane mentioned in BBAE I would remove the TRUE/FALSE if possible and just determine whether it's an email using a simple check such as '@'. Any option would work there.